Static type checking error

@pastorsj  Hi Sam!

I'm trying to pull series data from FRED and I'm running into a static type checking error. I'm not able to transpile when I'm using the wrapper syntax.

This is the callback I'm using:
```
async (req: Request, res: Response): Promise<void> => {
  try {
    const series = await fred.series.getSeries('SP500');

    res.send(series);

  } catch (err) {
    res.send(err);
  }
```
And this is the error I'm getting:
```
src/app.ts:14:46 - error TS2345: Argument of type 'string' is not assignable to parameter of type 'number'.

14   const series = await fred.series.getSeries('SP500');

```

I checked the wrapper documentation, and the `series_id` query parameter for getting an economic data series says that it's of type `string`, so there seems to be a disconnect here.

Thanks for looking into this! Let me know if I can help clarify anything.

--- a/index.d.ts
+++ b/index.d.ts
@@ -3,7 +3,7 @@
 // Definitions by: Sam Pastoriza <https://github.com/pastorsj>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export = Fred
+export = Fred;
 
 declare class Fred {
     constructor(apiKey: string, returnType?: string);
@@ -16,7 +16,6 @@ declare class Fred {
 }
 
 declare namespace FredAPI {
-    
     class Categories {
         getCategory(categoryId: number): Promise<CategorySet>;
         getChildCategories(categoryId: number, params?: object): Promise<CategorySet>;
@@ -25,7 +24,7 @@ declare namespace FredAPI {
         getCategoryTags(categoryId: number, params?: object): Promise<RelatedTagsInfo>;
         getCategoryRelatedTags(categoryId: number, params?: object): Promise<RelatedTagsInfo>;
     }
-    
+
     class Releases {
         getAllReleases(params?: object): Promise<ReleaseSet>;
         getAllReleasesWithDates(params?: object): Promise<ReleaseDateSet>;
@@ -37,200 +36,199 @@ declare namespace FredAPI {
         getRelatedTagsForRelease(releaseId: number, tagNames: string, params?: object): Promise<RelatedTagsInfo>;
         getTableTreesForRelease(releaseId: number, params?: object): Promise<TableTreesInfo>;
     }
-    
+
     class Series {
-        getSeries(seriesId: number, params?: object): Promise<SeriesSet>;
-        getCategoriesForSeries(seriesId: number, params?: object): Promise<CategorySet>;
-        getObservationsForSeries(seriesId: number, params?: object): Promise<ObservationInfo>;
-        getReleaseForSeries(seriesId: number, params?: object): Promise<ReleaseSet>;
+        getSeries(seriesId: string, params?: object): Promise<SeriesSet>;
+        getCategoriesForSeries(seriesId: string, params?: object): Promise<CategorySet>;
+        getObservationsForSeries(seriesId: string, params?: object): Promise<ObservationInfo>;
+        getReleaseForSeries(seriesId: string, params?: object): Promise<ReleaseSet>;
         getSeriesThatMatchesSearch(searchText: string, params?: object): Promise<SeriesInfo>;
         getTagsForSeriesSearch(seriesSearchText: string, params?: object): Promise<RelatedTagsInfo>;
         getRelatedTagsForSeriesSearch(seriesSearchText: string, params?: object): Promise<RelatedTagsInfo>;
-        getTagsForSeries(seriesId: number, params?: object): Promise<RelatedTagsInfo>;
+        getTagsForSeries(seriesId: string, params?: object): Promise<RelatedTagsInfo>;
         getUpdatedSeries(params?: object): Promise<UpdatedSeriesInfo>;
-        getUpdatedSeries(seriesId: number, params?: object): Promise<RevisedSeriesWithDates>;
+        getUpdatedSeries(seriesId: string, params?: object): Promise<RevisedSeriesWithDates>;
     }
-    
+
     class Sources {
         getAllSources(params?: object): Promise<SourcesInfo>;
         getSource(sourceId: number, params?: object): Promise<SourcesSet>;
         getReleasesForSource(sourceId: number, params?: object): Promise<ReleaseSet>;
     }
-    
+
     class Tags {
         getAllTags(params?: object): Promise<RelatedTagsInfo>;
         getAllRelatedTags(tagNames: string, params?: object): Promise<RelatedTagsInfo>;
         getAllSeriesMatchingTags(tagNames: string, params?: object): Promise<SeriesInfo>;
     }
-    
+
     interface TimeInfo {
-        realtime_start: string,
-        realtime_end: string,
+        realtime_start: string;
+        realtime_end: string;
     }
-    
+
     interface StandardInfo extends TimeInfo {
-        order_by: string,
-        sort_order: string,
-        count: number,
-        offset: number,
-        limit: number
+        order_by: string;
+        sort_order: string;
+        count: number;
+        offset: number;
+        limit: number;
     }
-    
-    /** 
+
+    /**
      *  -----------------------------------------------------
      *                   Category Typings
      *  -----------------------------------------------------
      */
-    
+
     interface CategorySet {
-        categories: Category[],
+        categories: Category[];
     }
-    
+
     interface RelatedTagsInfo extends StandardInfo {
-        tags: Tag[]
+        tags: Tag[];
     }
-    
+
     interface SeriesInfo extends StandardInfo {
-        series: SingularSeries[]
+        series: SingularSeries[];
     }
-    
+
     interface Category {
-        id: number,
-        name: string,
-        parent_id: number
+        id: number;
+        name: string;
+        parent_id: number;
     }
-    
+
     interface Tag {
-        name: string,
-        group_id: string,
-        notes: string,
-        created: string,
-        popularity: number,
-        series_count: number
+        name: string;
+        group_id: string;
+        notes: string;
+        created: string;
+        popularity: number;
+        series_count: number;
     }
-    
+
     interface SingularSeries {
-        id: string,
-        realtime_start: string,
-        realtime_end: string,
-        title: string,
-        observation_start: string,
-        observation_end: string,
-        frequency: string,
-        frequency_short: string,
-        units: string,
-        units_short: string,
-        seasonal_adjustment: string,
-        seasonal_adjustment_short: string,
-        last_updated: string,
-        popularity: number
+        id: string;
+        realtime_start: string;
+        realtime_end: string;
+        title: string;
+        observation_start: string;
+        observation_end: string;
+        frequency: string;
+        frequency_short: string;
+        units: string;
+        units_short: string;
+        seasonal_adjustment: string;
+        seasonal_adjustment_short: string;
+        last_updated: string;
+        popularity: number;
     }
-    
-    /** 
+
+    /**
      *  -----------------------------------------------------
      *                  Releases Typings
      *  -----------------------------------------------------
      */
-    
-     interface ReleaseInfo {
-         id: number,
-         realtime_start: string,
-         realtime_end: string,
-         name: string,
-         press_release: boolean,
-         link: string
-     }
-    
-     interface ReleaseSet extends TimeInfo {
-        releases: ReleaseInfo[]
-     }
-    
-     interface ReleaseWithDates extends StandardInfo {
-         release_dates: Releases[]
-     }
-    
-     interface BasicRelease {
-         release_id: number,
-         date: string
-     }
-    
-     interface SourcesSet extends TimeInfo {
-        sources: SingularSource[]
-     }
-    
-     interface SingularSource {
-         id: number,
-         realtime_start: string,
-         realtime_end: string,
-         name: string,
-         link: string
-     }
-    
-     interface TableTreesInfo {
-        name: string,
-        element_id: number,
-        release_id: number,
-        elements: object[]
-     }
-    
-     interface ReleaseSet extends StandardInfo {
-        releases: ReleaseInfo[]
-     }
-    
-     interface ReleaseDateSet extends StandardInfo {
-         release_dates: ReleaseWithDate[]
-     }
-    
-     interface ReleaseWithDate extends BasicRelease {
-         release_id: number,
-         date: string
-     }
 
-     /** 
+    interface ReleaseInfo {
+        id: number;
+        realtime_start: string;
+        realtime_end: string;
+        name: string;
+        press_release: boolean;
+        link: string;
+    }
+
+    interface ReleaseSet extends TimeInfo {
+        releases: ReleaseInfo[];
+    }
+
+    interface ReleaseWithDates extends StandardInfo {
+        release_dates: Releases[];
+    }
+
+    interface BasicRelease {
+        release_id: number;
+        date: string;
+    }
+
+    interface SourcesSet extends TimeInfo {
+        sources: SingularSource[];
+    }
+
+    interface SingularSource {
+        id: number;
+        realtime_start: string;
+        realtime_end: string;
+        name: string;
+        link: string;
+    }
+
+    interface TableTreesInfo {
+        name: string;
+        element_id: number;
+        release_id: number;
+        elements: object[];
+    }
+
+    interface ReleaseSet extends StandardInfo {
+        releases: ReleaseInfo[];
+    }
+
+    interface ReleaseDateSet extends StandardInfo {
+        release_dates: ReleaseWithDate[];
+    }
+
+    interface ReleaseWithDate extends BasicRelease {
+        release_id: number;
+        date: string;
+    }
+
+    /**
      *  -----------------------------------------------------
      *                  Series Typings
      *  -----------------------------------------------------
      */
 
-     interface SeriesInfo extends SingularSeries {
-        notes: string
-     }
+    interface SeriesInfo extends SingularSeries {
+        notes: string;
+    }
 
-     interface SeriesSet extends TimeInfo {
-         series: SeriesInfo[]
-     }
+    interface SeriesSet extends TimeInfo {
+        series: SeriesInfo[];
+    }
 
-     interface ObservationInfo extends StandardInfo {
-        observation_start: string,
-        observation_end: string,
-        units: string,
-        output_type: number,
-        file_type: string,
-        observations: ObservationSet
-     }
+    interface ObservationInfo extends StandardInfo {
+        observation_start: string;
+        observation_end: string;
+        units: string;
+        output_type: number;
+        file_type: string;
+        observations: ObservationSet;
+    }
 
-     interface ObservationSet extends TimeInfo {
-        date: string,
-        value: string
-     }
+    interface ObservationSet extends TimeInfo {
+        date: string;
+        value: string;
+    }
 
-     interface UpdatedSeriesInfo extends SeriesInfo {
-        filter_variable: string,
-        filter_value: string
-     }
+    interface UpdatedSeriesInfo extends SeriesInfo {
+        filter_variable: string;
+        filter_value: string;
+    }
 
-     interface RevisedSeriesWithDates extends StandardInfo {
-         vintage_dates: string[]
-     }
+    interface RevisedSeriesWithDates extends StandardInfo {
+        vintage_dates: string[];
+    }
 
-     /** 
+    /**
      *  -----------------------------------------------------
      *                  Sources Typings
      *  -----------------------------------------------------
      */
 
-     interface SourcesInfo extends StandardInfo {
-         sources: SingularSource[]
-     }
+    interface SourcesInfo extends StandardInfo {
+        sources: SingularSource[];
+    }
 }
-

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
     "name": "node-fred",
-    "version": "2.1.8",
+    "version": "2.1.9",
     "description": "Another nodejs wrapper for the st louis fed",
     "main": "lib/index.js",
     "types": "index.d.ts",

--- a/src/Api/series.js
+++ b/src/Api/series.js
@@ -9,7 +9,7 @@ class Series {
 
     /**
      * Gets an economic data series.
-     * @param {Number} seriesId
+     * @param {String} seriesId
      * @param {Object} params
      * @returns {Promise} Resolves with an economic data series or errors out
      */
@@ -39,7 +39,7 @@ class Series {
 
     /**
      * Gets the categories for an economic data series.
-     * @param {Number} seriesId
+     * @param {String} seriesId
      * @param {Object} params
      * @returns {Promise} Resolves with the categories for an economic data series or errors out
      */
@@ -69,7 +69,7 @@ class Series {
 
     /**
      * Gets the observations or data values for an economic data series.
-     * @param {Number} seriesId
+     * @param {String} seriesId
      * @param {Object} params
      * @returns {Promise} Resolves with the observations or data values
      * for an economic data series or errors out
@@ -110,7 +110,7 @@ class Series {
 
     /**
      * Gets the release for an economic data series.
-     * @param {Number} seriesId
+     * @param {String} seriesId
      * @param {Object} params
      * @returns {Promise} Resolves with the release for an economic data series or errors out
      */
@@ -179,7 +179,7 @@ class Series {
 
     /**
      * Gets the tags for a series search.
-     * @param {Number} seriesId
+     * @param {String} seriesId
      * @param {String} seriesSearchText
      * @param {Object} params
      * @returns {Promise} Resolves with the tags for a series search or errors out
@@ -217,7 +217,7 @@ class Series {
 
     /**
      * Gets the related tags for a series search.
-     * @param {Number} seriesId
+     * @param {String} seriesId
      * @param {String} seriesSearchText
      * @param {Object} params
      * @returns {Promise} Resolves with the related tags for a series search or errors out
@@ -256,7 +256,7 @@ class Series {
 
     /**
      * Gets the tags for an economic data series.
-     * @param {Number} seriesId
+     * @param {String} seriesId
      * @param {Object} params
      * @returns {Promise} Resolves with the tags for an economic data series or errors out
      */
@@ -321,7 +321,7 @@ class Series {
     /**
      * Gets the dates in history when a series' data values were
      * revised or new data values were released.
-     * @param {Number} seriesId
+     * @param {String} seriesId
      * @param {Object} params
      * @returns {Promise} Resolves with the dates in history when a
      * series' data values were revised or new data values were released or errors out

